encoding/codecv2/codecv2.go

```diff
@@ -315,11 +329,15 @@ func EstimateChunkL1CommitBatchSizeAndBlobSize(c *encoding.Chunk) (uint64, uint6
 	if err != nil {
 		return 0, 0, err
 	}
-	return uint64(len(batchBytes)), CalculatePaddedBlobSize(uint64(len(blobBytes))), nil
+	blobBytesLen := uint64(len(blobBytes))
+	if conditionalEncode {
+		blobBytesLen += 1
+	}
```

Comment:
I don't understand this, shouldn't it be something like this?

```go
if conditionalEncode {
    blobBytes, err := compressScrollBatchBytes(batchBytes)
    blobBytesLen = 1 + len(blobBytes)
} else {
    blobBytesLen = 1 + len(batchBytes)
}
```

Reply:
yeah. this should be fixed to align with compression behavior. previously I thought the only change was: that in blob bytes, there is only one extra flag to be added (but still missing some details), will refactor it to the new version.